python/tvm/relay/backend/executor_factory.py

```diff
@@ -60,37 +147,17 @@ def __init__(self, ir_mod, target, graph_json_str, libmod, libmod_name, params):
     def export_library(self, file_name, fcompile=None, addons=None, **kwargs):
         return self.module.export_library(file_name, fcompile, addons, **kwargs)
 
-    # Sometimes we want to get params explicitly.
-    # For example, we want to save its params value to
-    # an independent file.
+    def save_executor_config(self):
```

Comment:
seems like `get_executor_config` would be a better name since we are not actually writing to file here. alternatively, I think we want to eventually move export_model_library_format here, so could just continue using get_graph_json() conditionally there rather than introducing a new generic function here